server/routes/barcode/recipients/RecipientFormService.ts

```diff
@@ -67,7 +67,7 @@ export default class RecipientFormService {
   }
 
   private async toRecipient(recipientForm: RecipientForm): Promise<Recipient> {
-    const prison = await this.prisonRegisterService.getPrison(recipientForm.prisonId)
+    const prison = await this.prisonService.getPrison(recipientForm.prisonId)
```

Comment:
PrisonRegisterService is now hidden behind the facade PrisonService so it can apply supported prison logic globally.